monitoring/src/deposit-monitor.ts

```diff
@@ -7,30 +7,34 @@ import type { SystemEvent, Monitor as SystemEventMonitor } from "./system-event"
 import type { DepositRevealedEvent as DepositRevealedChainEvent } from "@keep-network/tbtc-v2.ts/dist/src/deposit"
 import type { Bridge } from "@keep-network/tbtc-v2.ts/dist/src/chain"
 
+function satsToRoundedBTC(sats: BigNumber): string {
+  return (sats.div(BigNumber.from(1e6)).toNumber() / 100).toFixed(2)
+}
+
 const DepositRevealed = (
   chainEvent: DepositRevealedChainEvent
 ): SystemEvent => ({
-  title: "Deposit revealed",
+  title: "Deposit Revealed",
   type: SystemEventType.Informational,
   data: {
-    btcFundingTxHash: chainEvent.fundingTxHash.toString(),
+    btcFundingTxHash: `https://mempool.space/tx/${chainEvent.fundingTxHash.toString()}`,
```

Comment:
Maybe add separate properties for the URLs to keep the `btcFundingTxHash` and `ethRevealTxHash` return the hashes?

Reply:
[80c599](https://github.com/keep-network/tbtc-v2/pull/612/commits/80c599af5e0e4ed094a39c6d52b94db6ef407950)